code/modules/uplink/uplink_items.dm

```diff
@@ -446,12 +444,19 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
 	item = /obj/item/guardiancreator/tech
-	cost = 15
+	cost = 12
 	manufacturer = /datum/corporation/traitor/cybersun
 	surplus = 0
-	//exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops) //yogs: removes restrictions on liabilit- I mean punchghosts
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```

Comment:
This change was not mentioned in the PR, why is this being removed from nukies?

Reply:
Look below - there's a new subtype exclusive to nukies that is 15 TC. The PR mentions that they cost 12 TC **for traitors**.

Reply:
https://github.com/yogstation13/Yogstation/blob/4f2d846351670abb14890e3e5fd9bd968faf6243/code/modules/uplink/uplink_items.dm#L455-L459